gql/filter.go

```diff
@@ -165,7 +165,15 @@ func newListFilter(arg interface{}) (*listFilter, error) {
 	return &listFilter{fieldName: lf.Field, op: op}, nil
 }
 
-func (lf listFilter) match(raw interface{}) bool {
-	field := DeepExtractField(raw, lf.fieldName)
-	return lf.op.Match(field)
+func (lf listFilter) match(raw interface{}) (bool, error) {
+	// In the case this is a limitLength filter, the fieldName is empty but DeepExtractField will always error.
```

Comment:
though limitLength is the only example of this now really any future filter with an empty fieldName is in the same boat, so really this is "DeepExtractField makes no sense when there is no fields specified"

Reply:
Cool, I'll update my comment since functionally it's the same